dissect/target/plugins/os/windows/amcache.py

```diff
@@ -607,6 +620,28 @@ def device_containers(self):
         if self.amcache:
             yield from self.parse_inventory_device_container()
 
+    @export(record=AppLaunchAppcompatRecord)
+    def applaunches(self):
+        """Return AppLaunchAppcompatRecord records from Amcache applaunch files (Windows 11 22H2 or later).
+
+        TODO: Research C:\\Windows\\appcompat\\pca\\PcaGeneralDb0.txt and
+              C:\\Windows\\appcompat\\pca\\PcaGeneralDb1.txt files.
+
+        Sources:
+            - https://aboutdfir.com/new-windows-11-pro-22h2-evidence-of-execution-artifact/
+        """
+
+        if (path := self.target.fs.path("sysvol/windows/appcompat/pca/PcaAppLaunchDic.txt")).exists():
+            for line in path.open("rt"):
+                if line.startswith("#") or line.strip() == "":
+                    continue
+                parts = line.rstrip().split("|")
+                yield AppLaunchAppcompatRecord(
+                    ts=datetime.strptime(parts[-1], "%Y-%m-%d %H:%M:%S.%f").replace(tzinfo=timezone.utc),
+                    path=parts[0],
```

Comment:
```suggestion
                    path=path.from_windows(parts[0]),
```
And need to add `from flow.record.fieldtypes import path`. You'd obviously have to rename the `path` variable in this function unfortunately.

Reply:
You will also need to add the import as mentioned in the comment.